util/SerializeUniverse.cpp

```diff
@@ -367,7 +367,7 @@ void serialize(Archive& ar, Fleet& obj, unsigned int const version)
     if (Archive::is_loading::value && version < 5) {
         bool aggressive = false;
         ar  & make_nvp("m_aggressive", aggressive);
-        obj.m_aggression = aggressive ? FleetAggression::FLEET_AGGRESSIVE : FleetAggression::FLEET_DEFENSIVE;
```

Comment:
this is fallback code for old versions of the serialization format, and the results should not change even if the UI behaviour or default when creating a new fleet changes

Reply:
I guess you are right, will revert that line.